.circleci/config.yml

```diff
@@ -22,6 +22,7 @@ jobs:
       - persist_to_workspace:
           root: .
           paths: "*"
+      - setup_remote_docker
```

Comment:
I think this needs to be a step inside the cluster-tests section, just below `- *fast-checkout`.